mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -1,19 +1,20 @@
 CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 	FIELD field_12730 listeners Ljava/util/List;
-	FIELD field_12731 buffer D
+	FIELD field_12731 safeZone D
 	FIELD field_12732 maxWorldBorderRadius I
 	FIELD field_12733 damagePerBlock D
 	FIELD field_12734 warningBlocks I
 	FIELD field_12735 warningTime I
 	FIELD field_12736 area Lnet/minecraft/class_2784$class_2785;
 	FIELD field_12737 centerZ D
 	FIELD field_12738 centerX D
+	FIELD field_24122 DEFAULT_INFO Lnet/minecraft/class_2784$class_5200;
```

Comment:
either stick with config, info, or properties. I'd support using config, so this one be
```suggestion
	FIELD field_24122 DEFAULT_CONFIG Lnet/minecraft/class_2784$class_5200;
```